contracts/generic/AugmintToken.sol

```diff
@@ -154,6 +154,10 @@ contract AugmintToken is AugmintTokenInterface {
         _transferFrom(from, to, amount, narrative);
     }
 
+    function setName(string _name) external restrict("StabilityBoard") {
```

Comment:
some comment with rational for this would be great. something like: "Allow Stability Board to change the name when a new token contract version is deployed and ready for production use. So that older token contracts are identifiable in 3rd party apps."